docs/cookbook/platform-sh.rst

```diff
@@ -242,7 +248,13 @@ When you get connected please run:
 
 .. code-block:: bash
 
-    $ php bin/console sylius:install
+    $ php bin/console sylius:install --env prod
+
+.. warning::
+
+    By default platform.sh creates only one instance of database with the `main` name. Installing Sylius on dev environment would required
+`main_dev` which of course will end up with an exception. Deploying development environment is possible but will require a little bit more
```

Comment:
Sure. I'm planning to add some more instructions to this document later based on my findings using Platform in production, but haven't had time to do it yet..